### PR TITLE
Add an option to run code examples in web containers

### PR DESCRIPTION
**Pull request by johnhwhite**

### CODE REVIEW

File: projects/docs-tools/src/modules/code-examples/code-examples-editor.service.spec.ts [hunk 1]
```diff
@@ -187,12 +187,7 @@ describe('Code examples editor service', () => {
   let service: SkyDocsCodeExamplesEditorService;
 
   beforeEach(() => {
-    moduleImports = [
-      'BrowserAnimationsModule',
-      'FormsModule',
-      'ReactiveFormsModule',
-      'RouterModule.forRoot([])',
-    ];
+    moduleImports = ['FormsModule', 'ReactiveFormsModule'];
```
Comment (johnhwhite) [inline projects/docs-tools/src/modules/code-examples/code-examples-editor.service.spec.ts:190]: I've removed `AppModule` and now bootstrap `AppComponent` as standalone. Theme and router move to `main.ts` and module imports move to `AppComponent`.

File: projects/docs-tools/src/modules/code-examples/code-examples-editor.service.ts [hunk 2]
```diff
@@ -105,9 +105,24 @@ export class SkyDocsCodeExamplesEditorService {
...
+          }
+        : undefined;
+
     const files = this.parseStackBlitzFiles(
       codeExample.sourceCode,
       mergedDependencies,
+      overrides,
+      {
+        target: template === 'node' ? 'ES2022' : 'ES2015',
+      },
```
Comment (johnhwhite) [inline projects/docs-tools/src/modules/code-examples/code-examples-editor.service.ts:125]: Web Container supports targeting ES2022, but the Engine Block requires ES2015.

File: projects/docs-tools/src/modules/code-examples/code-examples-editor.service.ts [hunk 3]
```diff
@@ -144,22 +161,23 @@ export class SkyDocsCodeExamplesEditorService {
...
+        {
+          env: {
+            NG_BUILD_PARALLEL_TS: '0',
+            NODE_ENV: 'development',
+          },
+        },
+        undefined,
+        2,
+      ),
+    };
```
Comment (johnhwhite) [inline projects/docs-tools/src/modules/code-examples/code-examples-editor.service.ts:180]: [Angular.dev](https://angular.dev/) uses these settings for their Web Containers.

File: projects/docs-tools/src/modules/demo-page/demo-page.component.spec.ts [hunk 4]
```diff
@@ -189,17 +189,11 @@ function getSidebarLinks(
...
 
   function setupTestBed(): void {
-    mockMediaQueryService = new MockSkyMediaQueryService();
     TestBed.configureTestingModule({
       imports: [DemoPageFixturesModule],
       providers: [
-        {
-          provide: SkyMediaQueryService,
-          useValue: mockMediaQueryService,
-        },
```
Comment (johnhwhite) [inline projects/docs-tools/src/modules/demo-page/demo-page.component.spec.ts:202]: Not used. Removing.

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Pull request merged**